src/types.ts

```diff
@@ -0,0 +1,25 @@
+import { BigNumber } from 'ethers';
+import { WalletType } from '@api3/operations';
+import { OpsGenieConfig, TelemetryChainConfig } from '@api3/operations-utilities';
+
+export interface WalletConfig {
```

Comment:
It might be worth converting these to Zod at some point, especially if this will be used by the public. I'll create an issue.

Reply:
Yeah great idea, it would greatly simplify the code as well. Thanks for creating the task :smile: